### PR TITLE
Updated alias imports

### PR DESCRIPTION
**Pull request by stijnbernards**

### CODE REVIEW

**Comment by stijnbernards** [comment]
The `ducks` naming almost caused me to commit `Fix fucks import` 😂

File: app/screens/MonthlyBudget/MonthlyBudgetScreen.tsx [hunk 1]
```diff
@@ -5,13 +5,17 @@ import { Ionicons } from "@expo/vector-icons";
...
 
+import { budget } from "ducks";
+
+import { t } from "utils";
+
+import { Colors } from "style";
+
+import { navigate } from "navigation";
+
 import { Text, Button } from "../../components";
```
Comment (PierreBresson) [inline app/screens/MonthlyBudget/MonthlyBudgetScreen.tsx:16]: Should be `import { Text, Button } from "components";`

**Review by PierreBresson — CHANGES_REQUESTED** [review]
You've missded all`components` imports

File: app/screens/MonthlyBudget/MonthlyBudgetScreen.tsx (hunk 1 above)
Comment (stijnbernards) [reply app/screens/MonthlyBudget/MonthlyBudgetScreen.tsx:16]: I'll take a look at this tonight or tomorrow :)

**Comment by PierreBresson** [comment]
Done https://github.com/NMF-earth/nmf-app/issues/177

**Pull request closed without merging**